mobile-center

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
It's been removed and replaced by the auto-generated npm launch script. I'd rather remove the ad-hoc launch scripts from the repo. For dev purposes we can take the hit to run node/ts-node directly, but it's less confusing for our customers, and easier for us to maintain installation/package publishing, if we have one canonical way to launch the thing.